include/nbl/video/SPhysicalDeviceFeatures.h

```diff
@@ -0,0 +1,511 @@
+#ifndef _NBL_VIDEO_S_PHYSICAL_DEVICE_FEATURES_H_INCLUDED_
+#define _NBL_VIDEO_S_PHYSICAL_DEVICE_FEATURES_H_INCLUDED_
+
+namespace nbl::video
+{
+
+struct SPhysicalDeviceFeatures
```

Comment:
at the end, do a finall pass over all the fields and double check if they're really Features (can be enabled/disabled in Vulkan) or just Limits